conda.recipe/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "dev" %}
+
+package:
+  name: udunits2
+  version: {{ version }}
+
+source:
+  path: ../
+
+build:
+  detect_binary_files_with_prefix: true
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - python  # [win]
```

Comment:
This Python dependency is only so `conda` know to activate the correct `vc`. Udunits2 does not depend on Python!